crates/usvg/src/tree/text.rs

```diff
@@ -562,12 +567,20 @@ impl Text {
     }
 
     /// Text converted into paths, ready to render.
-    ///
-    /// Returns `None` when the `text` build feature was disabled.
     pub fn flattened(&self) -> &Group {
         &self.flattened
     }
 
+    /// The positioned glyphs and decoration spans of the text.
+    ///
+    /// This should only be used if you need more low-level access
+    /// to the glyphs that make up the text. If you just need the
+    /// outlines of the text, you should use `flattened` instead.
+    #[cfg(feature = "text")]
+    pub fn layouted(&self) -> &Vec<TextFragment> {
```

Comment:
Shouldn't we return just `[TextFragment]`?

Reply:
Makes sense. 😄